docs/solution-guidance/search-solutions-in-sharepoint-2013-and-sharepoint-online.md

```diff
@@ -317,14 +317,14 @@ The next element is a `div` element. This is where you place any HTML or script
 
 The following are some additional resources for display templates and result types:
 
-- [Customize search result types in SharePoint Server](https://docs.microsoft.com/SharePoint/search/customize-search-result-types)
-- [How to change the way search results are displayed in SharePoint Server](https://docs.microsoft.com/SharePoint/search/how-to-change-the-way-search-results-are-displayed)
-- [Understanding how search results are displayed in SharePoint Server](https://docs.microsoft.com/SharePoint/search/understanding-how-search-results-are-displayed)
-- [Understanding how item display templates and hit highlighting work in SharePoint Server](https://docs.microsoft.com/SharePoint/search/understanding-how-item-display-templates-and-hit-highlighting-work)
-- [How to create a new result type in SharePoint Server](https://docs.microsoft.com/SharePoint/search/how-to-create-a-new-result-type)
-- [How to display values from custom managed properties in search results - option 1 in SharePoint Server](https://docs.microsoft.com/SharePoint/search/how-to-display-values-from-custom-managed-properties-in-search-resultsoption-1)
-- [How to display values from custom managed properties in search results - option 2 in SharePoint Server](https://docs.microsoft.com/SharePoint/search/how-to-display-values-from-custom-managed-properties-in-search-resultsoption-2)
-- [How to display values from custom managed properties in the hover panel in SharePoint Server](https://docs.microsoft.com/SharePoint/search/how-to-display-values-from-custom-managed-properties-in-the-hover-panel)
+- [Customize search result types in SharePoint Server](/SharePoint/search/customize-search-result-types)
+- [How to change the way search results are displayed in SharePoint Server](/SharePoint/search/how-to-change-the-way-search-results-are-displayed)
+- [Understanding how search results are displayed in SharePoint Server](/SharePoint/search/understanding-how-search-results-are-displayed)
+- [Understanding how item display templates and hit highlighting work in SharePoint Server](/SharePoint/search/understanding-how-item-display-templates-and-hit-highlighting-work)
+- [How to create a new result type in SharePoint Server](/SharePoint/search/how-to-create-a-new-result-type)
+- [How to display values from custom managed properties in search results - option 1 in SharePoint Server](/SharePoint/search/how-to-display-values-from-custom-managed-properties-in-search-resultsoption-1)
+- [How to display values from custom managed properties in search results - option 2 in SharePoint Server](/SharePoint/search/how-to-display-values-from-custom-managed-properties-in-search-resultsoption-2)
+<!--- - [How to display values from custom managed properties in the hover panel in SharePoint Server](SharePoint/search/how-to-display-values-from-custom-managed-properties-in-the-hover-panel) --->

```

Comment:
What's the point of this change? If the content shouldn't be the there, remove it... don't comment it out.